model-fcrepo/src/main/java/edu/unc/lib/boxc/model/fcrepo/objects/WorkObjectImpl.java

```diff
@@ -214,4 +202,26 @@ public List<PID> getMemberOrder() {
                 .map(PIDs::get)
                 .collect(Collectors.toList());
     }
+
+    @Override
+    public FileObject getThumbnailObject() {
+        return getObject(Cdr.useAsThumbnail);
+    }
+
+    private FileObject getObject(Property property) {
```

Comment:
how about `getFileObjectByProperty` or relation? `getObject` is very generic in java (everything in java is an Object, except for a few things like ints)